lib/LambdaHelper.js

```diff
@@ -68,7 +65,13 @@ var _lambdaize = function (userFunc) {
             func.apply(this, event.args);
         }
         catch(error) {
-            _newCallback({isFaulty: true, stack: error.stack, message: error.message});
+            var objectifyError = function(err) {
```

Comment:
No that is the purpose of it; there can be only one error therefore we
declare it only if it happens (very small optimization but why not)

Sent from my Nexus
On Jan 27, 2015 11:00 AM, "EFF" <notifications@github.com> wrote:

> In lib/LambdaHelper.js
> <https://github.com/mentum/lambdaws/pull/53#discussion_r23618068>:
>
> > @@ -68,7 +65,13 @@ var _lambdaize = function (userFunc) {
> >              func.apply(this, event.args);
> >          }
> >          catch(error) {
> > -            _newCallback({isFaulty: true, stack: error.stack, message: error.message});
> > +            var objectifyError = function(err) {
>
> Should this function be outside the catch block so it doesn't get
> instantiated every time there is an error ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mentum/lambdaws/pull/53/files#r23618068>.
>